packages/eslint-config-angular/test/member-ordering-correct.test.js

```diff
@@ -0,0 +1,42 @@
+import ESlint from 'eslint';
+import path from 'path';
+
+const rule = '@typescript-eslint/member-ordering';
+
+describe('@tinkoff/eslint-config-angular', () => {
+    describe(rule, () => {
+        const cli = new ESlint.CLIEngine({
+            cwd: path.join(__dirname, '..'),
+            useEslintrc: false,
+            baseConfig: {
+                parserOptions: require('../index.js').parserOptions,
+                overrides: [
```

Comment:
Aren't these overrides need to be directly in rules/typescript-eslint-plugin?

Reply:
Yes, but I only want to test `member ordering` rules in isolation without side effect with another rules